.travis.yml

```diff
@@ -26,6 +26,27 @@ matrix:
         - libhwloc-dev
         - uuid-dev
         - doxygen
+  - script: scripts/build-pypi.sh
+    install:
+    - pip install --user pybind11 nose2
+    cache: ccache
+    env:
+    - TRAVIS_BUILD_TYPE=RelWithDebInfo
+    addons:
+      apt:
+        packages:
+        - libjson-c-dev
+        - libhwloc-dev
+        - uuid-dev
+        - doxygen
+    deploy:
+      provider: pypi
+      user: opae_robot
+      password:
+         secure: "nbiIOg2D8zYuQaSBuVNam5ga3OORqukpFYl8W/uV5r5rh81TwhsY0Y45HLxviRRUK/5/cBgVxfXDzKlzJSG0LDW3zz9ivaAihVARRmOlyCN9akvx6XnAE4kJ/al4qdsC4u0rOUWkS9FwEqlhF6sgpDiJzD/USjItkDKnWampU3j9jW85as52jngP8f8DKgL0k6lOXi//AfdZSdsXE5sSUUx1DDOk5ta3SL+Qeme2C46bnjkPLiJusPn76AfB1IcfcgNzHtieLkR/LRe3rewmG7u/GRlCWkNRYHeXfPdqAHizN+wE68ypdVRKWS3eUtAu2X3Y/Dc0+VrGR7zVpqb8YJNxZ7kmG05wWrF6Gb6KGD3bu59CnSKNvELFgZxZrGyYNJdX5w9qLdWE1I05bKvi5bFGYtxzaCLV9lIEuhQegYW+ADFki8n9/edUPnV7choobfKRluiVIi0q2pbQKxqZKSQ8JBrH8Jo5AIqAxv43PmbU+eHmxS9iyNsUtxTns2vuPr+dmO51JfdufkWLrxZDN7Ygz2Z71z7fCrb/BPQ2oU6fnamr3kco3eLBcgEiLVbcBIjNPI2rCjJFN4Tgs63nwqVFVZciOH3IFRcNi0S8Nxbb8C3QnD9DLffv23n9XNI8ffJ1fjp+bpRuZ61HMktuKR5dV9mqivFCF98TSd0O5Xc=" 
+      distributions: sdist
+      on:
+        tags: true
```

Comment:
Does this only upload for tags?

Reply:
Yes. Building is done with each build in order to verify if there are any issues but uploading is only done if there is a new tag.